msfs-avionics-mirror/src/sdk/.npmrc

```diff
@@ -0,0 +1 @@
+package-lock=false
```

Comment:
As we've switched the main repository, how about we now also generate a package-lock.json here?